flexmeasures/ui/__init__.py

```diff
@@ -57,6 +58,9 @@ def favicon():
     add_jinja_filters(app)
     add_jinja_variables(app)
 
+    # re-register api blueprint so it'll register the chart views (see views.charts)
```

Comment:
`views.charts` doesn't explain why we need to re-register as far as I can tell.

Reply:
That refers to a unique way of working in Flask. We need to call this method on a blueprint again, if since the first registration we imported new views which want to attach their route to that blueprint. It's not documented, as it is a bit unusual, but luckily I found this parameter, which led me to assume this.